test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Command.Tests.ps1

```diff
@@ -103,3 +103,79 @@ Describe "Get-Command Feature tests" -Tag Feature {
         }
     }
 }
+
+Describe "Get-Command" -Tag CI {
+    BeforeAll {
+        Import-Module Microsoft.PowerShell.Management
+    }
+    Context "-Syntax tests" {
+        It "Should return a string object when -Name is an alias and -Syntax is specified" {
+            $Result = Get-Command -Name del -Syntax
+
+            $Result | Should -BeOfType [String]
+            $Result | Should -Match 'del \[-Path\]'
+        }
+
+        It "Should replace commands with aliases in matching commands when using a wildcard search" {
+            $Result = Get-Command -Name sp* -Syntax
+
+            $Result | Should -BeOfType [String]
+            $Result -join '' | Should -Match 'sp \(alias\) -> Set-ItemProperty'
+            $Result -join '' | Should -Match 'sp \[-Path\]'
+        }
+
+        It "Should not add the alias (alias) -> command decorator for non-alias commands" {
+            $Result = Get-Command -Name sp* -Syntax
+
+            $Result -join '' | Should -Not -Match 'Split-Path \(alias\) -> Split-Path'
+            $Result -join '' | Should -Match 'Split-Path \[-Path\]'
+        }
+
+        It "Should only replace aliases when given multiple entries including a command and an alias" {
+            $Result = Get-Command -Name get-help, del -Syntax
+
+            $Result -join '' | Should -Match 'del \(alias\) -> Remove-Item'
+            $Result -join '' | Should -Match 'del \[-Path\]'
+            $Result -join '' | Should -Match 'Get-Help \[\[-Name\]'
+            $Result -join '' | Should -Not -Match 'del \(alias\) -> Get-Help'
+        }
+
+        It "Should return the path to an aliased script when -Syntax is specified" {
+            # First, create a script file
+            @'
+            [CmdletBinding()]
+            param(
+                [Parameter(Position=0, Mandatory, ValueFromPipeline, ValueFromPipelineByPropertyName)]
+                [ValidateNotNullOrEmpty()]
+                [string[]]
+                $Name
+            )
+            process {
+                "Processing ${Name}"
+            }
+'@ | out-File TestDrive:\Test-GcmSyntax.ps1
```

Comment:
NIT: Personally, would prefer the herestring to be assigned to a variable and use `Set-Content` on a different line as I think it's more readable

```suggestion
'@ | Out-File -Path TestDrive:\Test-GcmSyntax.ps1
```

Reply:
Fixed.